frontend/src/modules/S3_Datasets/views/DatasetView.js

```diff
@@ -161,7 +161,7 @@ const DatasetView = () => {
   const fetchItem = useCallback(async () => {
     setLoading(true);
     const response = await client.query(getDataset(params.uri));
-    if (!response.errors && response.data.getDataset !== null) {
+    if (response.data.getDataset !== null) {
```

Comment:
nit: expose all errors

Reply:
Same as with Dashboards. I don't know if adding more error banners is going to give users more useful info. If the top level query fails, then the errors that we might find in the nested resolvers are very unpredictable. For example if get_Dataset fails because the dataset is not found, the restricted is going to fail with something as 'URI is null'; which might be misleading.

Reply:
is it guaranteed the first error in the dispatch always the top level?